tests/System.Text.Json.Tests/JsonWriterPerfTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+using System.Text.Formatting;
+using System.IO;
+
+namespace System.Text.Json.Tests
+{
+    public partial class JsonPerfTests
```

Comment:
I think it would be good to add a test that simulates what's happening in JSON TechEmpower, i.e. write {"message":"Hello, World!"}